modules/runner/src/integrationTest/java/org/apache/ignite/internal/sql/engine/ItUuidTest.java

```diff
@@ -158,9 +159,9 @@ public void testBasicAggregates() {
     public void testUuidCaseExpression() {
         UUID other = UUID.randomUUID();
 
-        sql("INSERT INTO t VALUES (1, ?)", UUID_1);
-        sql("INSERT INTO t VALUES (2, ?)", UUID_2);
-        sql("INSERT INTO t VALUES (3, NULL)");
+        sql("INSERT INTO t (id, uuid_key) VALUES (1, ?)", UUID_1);
+        sql("INSERT INTO t (id, uuid_key) VALUES (2, ?)", UUID_2);
+        sql("INSERT INTO t (id, uuid_key) VALUES (3, NULL)");
```

Comment:
I would prefer to have a single INSERT statement for the three values, like `INSERT INTO t (id, uuid_key) VALUES (1, ?), (2,?), (3,?)`, it could catch additional problems.

Reply:
Simplified to a single INSERT statement.